src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs

```diff
@@ -30,10 +30,19 @@ namespace Microsoft.Extensions.DependencyInjection
     public static class MvcCoreServiceCollectionExtensions
     {
         /// <summary>
-        /// Adds essential MVC services to the specified <see cref="IServiceCollection" />.
+        /// Adds the minimum essential MVC services to the specified <see cref="IServiceCollection" />. Addtional services
+        /// including MVC's support for authorization, formatters, and validation must be added separate using the 
+        /// <see cref="IMvcCoreBuilder"/> returned from this method.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
         /// <returns>An <see cref="IMvcCoreBuilder"/> that can be used to further configure the MVC services.</returns>
+        /// <remarks>
+        /// The <see cref="AddMvcCore"/> approach for configuring MVC is provided for experienced MVC developers who wish 
+        /// to have full control over the set of default services registerred. As such, we register the minimum set of
```

Comment:
In terms of style, we recommend avoiding "we" because it's not "us" doing anything, it's the code. So maybe something like:

`As such, only the minimum set of services necessary to do X, Y, and Z are registered.`